core/api/service/chain/chain_jrpc_processor.cpp

```diff
@@ -31,6 +33,12 @@ namespace kagome::api::chain {
     server_->registerHandler("chain_getHeader",
                              Handler<request::GetHeader>(api_));
 
+    server_->registerHandler("chain_subscribeFinalizedHeads",
+                             Handler<request::SubscribeFinalizedHeads>(api_));
+
+    server_->registerHandler("chain_unsubscribeFinalizedHeads",
+                             Handler<request::UnsubscribeFinalizedHeads>(api_));
```

Comment:
Please register aliases `chain_subscribeFinalizedHead` and `chain_unsubscribeFinalizedHead` the same way as aliases for `*NewHead` rpcs?